CONTRIBUTING.md

```diff
@@ -83,22 +83,25 @@ bundle exec rubocop --auto-gen-config
 
 ### Playground <!-- omit in toc -->
 
-First, you need to run a Meilisearch instance:
+Check the [playground's README](./playground/README.md) for more information.
+
+Use docker to run the playground environment:
 
 ```bash
-docker run -p 7700:7700 getmeili/meilisearch:latest meilisearch --no-analytics
+docker-compose up playground
 ```
 
-To test directly your changes in `meilisearch-rails`, you can run the Rails playground:
+It will use your local `meilisearch-rails` source code as the `meilisearch-rails` gem.
+And will get the same `meilisearch` instance running as the `package` service.
+
+#### Troubleshoot:
+
+If for some reason the app does not start open a `playground` console with `docker-compose run --rm playground bash` and run these administrative commands:
```

Comment:
`these administrative commands`
This is the first time I have seen this expression 😃 Does this mean something in particular? I tried to translate it to french but the translation seems weird 👀